scripting/get5.sp

```diff
@@ -231,8 +231,8 @@ public void OnPluginStart() {
         "Maximum number of pauses a team can use, 0=unlimited");
     g_MaxPauseTimeCvar = CreateConVar("get5_max_pause_time", "300",
         "Maximum number of time the game can spend paused by a team, 0=unlimited");
-    g_QuickRestartCvar = CreateConVar("get5_quick_restarts", "0",
-        "Whether to use a quick restart or a full live-on-3 restart. 0 for LO3.");
+    g_QuickRestartCvar = CreateConVar("get5_use_lo3", "0",
+        "Whether to use a quick restart or a full live-on-3 restart.");
```

Comment:
I don't think a "quick restart" really describes what happens when this is 0. "Whether to do a live-on-3 restart in addition to the warmup countdown" would be better.